.binder/requirements.txt

```diff
@@ -1 +1,2 @@
 PyMPDATA-examples
+imageio
```

Comment:
this should not be needed if we have it in setup.py requirements, right?
If it is needed, then we should also add it to Colab-related install instructions.